css/properties.json

```diff
@@ -4641,30 +4641,40 @@
     "status": "standard"
   },
   "offset": {
-    "syntax": "&lt;'offset-path'&gt; &amp;&amp; &lt;'offset-distance'&gt; &amp;&amp; &lt;'offset-rotation'&gt; &amp;&amp; &lt;'offset-anchor'&gt;",
+    "syntax": "[ &lt;'offset-position'&gt;? [ &lt;'offset-path'&gt; [ &lt;'offset-distance'&gt; || &lt;'offset-rotate'&gt; ]? ]? ]! [ / &lt;'offset-anchor'&gt; ]?",
     "media": "visual",
     "inherited": false,
-    "animationType": "discrete",
+    "animationType": [
+      "offset-position",
+      "offset-path",
+      "offset-distance",
+      "offset-anchor",
+      "offset-rotate"
+    ],
     "percentages": [
+      "offset-position",
       "offset-distance"
+      "offset-anchor",
```

Comment:
That's a syntax error. The comma needs to be moved to `"offset-distance"`.

Sebastian